modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -6891,6 +6900,7 @@ existing-ratings-data-values-will-be-adapted-to-match-the-new-ratings-type-even-
 existing-structure=Existing Structure
 existing-tokens=Existing Tokens
 exit=Exit
+exit-with-primary-unit-of-measure-changed-saving-help=If the actual unit of measure is changed, all conversion rates to primary units of measure already set may become incorrect. Are you sure you want to change the Primary UOM ?
```

Comment:
I think it has an extra space at the end before the question mark, can you remove it?